src/fciqmc.jl

```diff
@@ -188,4 +188,18 @@ end
 function advance!(algorithm, report, state::ReplicaState, replica::SpectralState{1})
     return advance!(algorithm, report, state, only(replica.single_states))
 end
-# TODO: add advance! for SpectralState{N} where N > 1
+
+function advance!(algorithm, report, state::ReplicaState, replica::SpectralState{N, NS, GramSchmidt{N}}) where {N, NS}
```

Comment:
```suggestion
function advance!(algorithm, report, state::ReplicaState, replica::SpectralState{N, <:Any, GramSchmidt{N}}) where {N}
```
I don't think that we need `NS`.